proto/sentry_protos/snuba/v1/endpoint_delete_trace_items.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/request_common.proto";
+// import "sentry_protos/snuba/v1/trace_item_filter.proto";
+
+// DeleteTraceItemsRequest initiates a delete of all matching items in a trace.
+// It should also remove downsampled copies.
+//
+// The response will be an acknowledgement of a request, but not
+// a guarantee that the delete has been fulfilled.
+message DeleteTraceItemsRequest {
+  RequestMeta meta = 1;
+  repeated string trace_ids = 2;
+  TraceItemFilterWithType filter = 3;
```

Comment:
I think this should be a repeated field as we may need to specify several item types with conditions.